### PR TITLE
Fixed typos in README.md

### PR DESCRIPTION
**Pull request by arnoldstoba**

### CODE REVIEW

**Comment by arnoldstoba** [comment]
**Edit:** Signed CLA

File: README.md [hunk 1]
```diff
@@ -90,13 +90,13 @@ Let's break down the parts of this message.
 * `[package.Type]`: The package.Type of the modifier that you want to use. In
   this case, it's "header.Modifier", which is the name of the modifier that
   sets headers (to learn more about the `header.Modifier`, please
-  refer to the [modifier reference](https://github.com/google/martian/wiki/Modifier-Reference).
+  refer to the [modifier reference](https://github.com/google/martian/wiki/Modifier-Reference)).
```
Comment (bramhaghosh) [inline README.md]: extra closing paren is not neccessary

Comment (arnoldstoba) [inline README.md]: Well, the sentence starts after a bracket so there should be a closing bracket somewhere.

Comment (bramhaghosh) [inline README.md]: ooooohhhh - ok I see it now - thanks!

Comment (bramhaghosh) [inline README.md]: I see it now! good catch - thanks!!

**Pull request merged**